src/python/foglamp/device/__main__.py

```diff
@@ -16,25 +18,19 @@
 __license__ = "Apache 2.0"
 __version__ = "${VERSION}"
 
+_logger = logger.setup("Device", level=20)
 
-# TODO: Support --help, --name plugin, etc.
-try:
-    plugin = sys.argv[1]
-    index = plugin.find('--name=')
-    if index == -1:
-        plugin = None
-    else:
-        plugin = plugin
-except IndexError:
-    plugin = None
 
-if plugin is None:
-    # TODO: delete this
-    plugin = 'CoAP'
+plugin = Parser.get('--name')
+core_mgt_port = Parser.get('--port')
+core_mgt_address = Parser.get('--address')
 
 if plugin is None:
-    # TODO: Output to STDERR
-    print("Required argument 'name' is missing")
-    # TODO: exit 1
+    print("Required argument '--name' is missing")
```

Comment:
maybe it does (so please ignore the msg), this should go to syslog, and so the other print(s)